Sources/TuistDependencies/Carthage/Generators/CartfileContentGenerator.swift

```diff
@@ -0,0 +1,66 @@
+import TSCBasic
+import TuistCore
+import TuistSupport
+
+// MARK: - Cartfile Content Generating Error
+
+enum CartfileContentGeneratorError: FatalError, Equatable {
```

Comment:
Once you introduce a requirement type specific to Carthage, I think you can remove this file entirely because we no longer need to validate.